Build broken due to string enum member check failing

### Description

All builds (for all open PRs) are currently broken because of a TypeScript build error in the `@slack/rtm-api` package:

```
src/RTMClient.ts:136:74 - error TS2345: Argument of type 'string' is not assignable to parameter of type 'UnrecoverableRTMStartError'.

136                       Object.values(UnrecoverableRTMStartError).includes(error.data.error)) {
```

I'm still gathering information to understand the cause, but I'll share what I know so far. The builds were passing on my local machine before I cleaned my working copy (by removing all `node_modules` directories and all `package-lock.json` files). After that, I got the same errors locally.

I also investigated the difference in how the tests are invoked on CI versus locally. Specifically, on CI the `--no-ci --hoist` flags are passed to `lerna bootstrap`. I can confirm that those flags have nothing to do with this issue. The `--no-ci` flag makes the CI environment work more like a local environment, and the `--hoist` flag moves common dependencies to the root `node_modules` directory. Since the errors occurred without either of those flags locally, its safe to conclude that the error is not related to them.

My next hunch is that it has to do with a later version of `typescript` being installed. The version that is installed across all packages locally when I reproduced the error is `3.6.3`, while the `package.json` files specify a range of `^3.3.3333`. It is possible that the semantics of a string enum changed in TypeScript releases 3.4,  3.5, or 3.6. That's what I'll be investigating next.


### Requirements (place an `x` in each of the `[ ]`)
* [x] I've read and understood the [Contributing guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

--- a/packages/rtm-api/src/RTMClient.ts
+++ b/packages/rtm-api/src/RTMClient.ts
@@ -133,7 +133,7 @@ export class RTMClient extends EventEmitter {
 
                   let isRecoverable = true;
                   if (error.code === APICallErrorCode.PlatformError &&
-                      Object.values(UnrecoverableRTMStartError).includes(error.data.error)) {
+                      (Object.values(UnrecoverableRTMStartError) as string[]).includes(error.data.error)) {
                     isRecoverable = false;
                   } else if (error.code === APICallErrorCode.RequestError) {
                     isRecoverable = false;

